vite.config.ts

```diff
@@ -1,7 +1,36 @@
-import { defineConfig } from "vite";
+import { defineConfig, loadEnv } from "vite";
 import react from "@vitejs/plugin-react";
 
+function getFriendbotUrl(mode: string) {
+  // See https://developers.stellar.org/docs/tools/quickstart/faucet for more information
+  // on friendbot and the local faucet
+  const env = loadEnv(mode, process.cwd(), '')
+  switch (env.PUBLIC_STELLAR_NETWORK) {
+    case "local":
+      return "http://localhost:8000";
+    case "testnet":
+    case "futurenet":
+      return "https://friendbot.stellar.org";
```

Comment:
```suggestion
    case "futurenet":
      return "https://friendbot-futurenet.stellar.org";
    case "testnet":
      return "https://friendbot.stellar.org";
```

Took a while to track this down, but you can find this info at https://developers.stellar.org/docs/networks